SimGeneral/MixingModule/python/digitizers_cfi.py

```diff
@@ -104,10 +106,9 @@
 
 from Configuration.ProcessModifiers.run3_ecalclustering_cff import run3_ecalclustering
 (run3_ecalclustering | phase2_hgcal).toModify( theDigitizersValid,
```

Comment:
this could just be `run3_ecalclustering` now (`phase2_hgcal` modification is now redundant)